ddtrace/contrib/elasticsearch/patch.py

```diff
@@ -14,6 +15,12 @@
 DEFAULT_SERVICE = 'elasticsearch'
 SPAN_TYPE = 'elasticsearch'
 
+# Default settings
+config._add(
+    DEFAULT_SERVICE,
+    {'service_name': DEFAULT_SERVICE},
+)
```

Comment:
This is tricky. Some modules use constants. So if those constants are changed, downstream users will suddenly break. And that dependency most likely won't be so clear to maintainers/contributors who might want to change the constant values.